### ISSUE TITLE
False positive Z309 for keys

### ISSUE THREAD

**Issue by sobolevn** [wemake-services/wemake-python-styleguide#528]
# Bug report

## What's wrong

```python
def test(office, returned_item):
    assert returned_item['id'] == office.id
```

raises `Z309 Found reversed comparison order` violation

## How is that should be

It should not raise a violation. We use variables on both sides.

## System information

I am using `0.8.1` release.
